### PR TITLE
Chore: Export new icon

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

New icon is not exported and can't be used.

# Changes

* Add new icon to export list.

# Screenshots

No UI changes

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd please review

File: src/lib/icons.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export { default as IconClockNoFill } from "./icons/IconClockNoFill.svelte";
 export { default as IconClose } from "./icons/IconClose.svelte";
 export { default as IconCopy } from "./icons/IconCopy.svelte";
 export { default as IconDarkMode } from "./icons/IconDarkMode.svelte";
+export { default as IconDisolving } from "./icons/IconDissolving.svelte";
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply src/lib/icons.ts:12]: yes, thanks!

[the remark this answers is not in the document]
Comment (lmuntaner) [reply src/lib/icons.ts]: done

**Comment by lmuntaner** [comment]
@dskloetd ready for review

**Review by dskloetd — APPROVED** [review]

**Pull request merged**